app/models/compute_resource.rb

```diff
@@ -183,6 +183,8 @@ def supports_vms_pagination?
 
   def find_vm_by_uuid(uuid)
     client.servers.get(uuid) || raise(ActiveRecord::RecordNotFound)
+  rescue
```

Comment:
Ouch, this will swallow all errors which is generally not a good idea. If you know the implications, _really_ want and need to do this, please log the exception.
```suggestion
  rescue StandardError => e
    Foreman::Logging.exception('Some meaningful description', e)
```